recipes/hiplot/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "hiplot" %}
+{% set version = "0.1.5" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: ad984b72b98c6adbfba958d897ff42738882e1cd07b8f4f835f3f845fb147803
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  noarch: python
+
+requirements:
+  host:
+    - beautifulsoup4
```

Comment:
Usually `pip` and `python` are the only dependencies that should go in `host`, and everything else goes to `run`. Can you please try that?